manifests/plugin/ansible.pp

```diff
@@ -21,6 +21,9 @@
 #                      environments like Foreman Remote Execution or cron
 #
 # $stdout_callback:: Ansible's stdout_callback setting
+#
+# $roles_path:: Paths where we look for ansible roles.
+
```

Comment:
I think this empty line may have broken things. Oddly enough our linter is flexible enough that it didn't catch it but puppet-strings is a bit more strict